HintList.py

```diff
@@ -1232,22 +1251,179 @@ def tokens_required_by_settings(world):
+    '1210':                                                     ("Did you know that you can Boomerang items that are freestanding Heart Pieces in the unrandomized game?", None, 'junk'),
+    '1211':                                                     ("Did you know that ReDead won't attack if you walk very slowly?", None, 'junk'),
+    '1212':                                                     ("Did you know that ReDead and Gibdo have their own version of Sun's Song that freezes you?", None, 'junk'),
+    '1213':                                                     ("${12 28 B1}\u009F \u00A7\u00A8\u00A6 \u00A7\u00A8\u00A6 \u009F\u00A6 \u009F\u00A6 \u00A8\u00A7\u009F", None, 'junk'), # ref: Frogs 2 / sfx: Frogs
+    '1214':                                                     ("${12 28 A2}Help! I'm melting away!", None, 'junk'), # sfx: red ice melting
+    '1215':                                                     ("${12 38 80}Eek!^I'm a little shy...", None, 'junk'), # sfx: Scrub hurt/stunned by Link
+    '1216':                                                     ("Master, there is a 0 percent chance that this hint is useful in any way.", None, 'junk'),
+    '1217':                                                     ("${12 48 0B}Here, have a heart <3", None, 'junk'), # sfx: get Recovery Heart
+    '1218':                                                     ("${12 48 03}Here, have a Rupee.", None, 'junk'), # sfx: get Rupee
+    '1219':                                                     ("${12 68 31}Don't forget to stand up and stretch regularly.", None, 'junk'), # sfx: child Link stretching and yawning
+    '1220':                                                     ("Remember that time you did that really embarrassing thing?^${12 68 3A}Yikes.", None, 'junk'), # sfx: child Link fall damage
+    '1221':                                                     ("@ tries to read the Gossip Stone...^${12 48 06}but he's standing on the wrong side of it!", None, 'junk'), # ref: Dragon Quest XI / sfx: error (e.g. trying to equip an item as the wrong age)
+    '1222':                                                     ("Plandomizer is a pathway to many abilities some consider to be... unnatural.", None, 'junk'),
+    '1223':                                                     ("Did you know that you can have complete control over the item placement, item pool and more using Plandomizer?", None, 'junk'),
+    '1224':                                                     ("They say that the earth is round.^Just like pizza.", None, 'junk'),
+    '1225':                                                     ("${12 68 62}Keeeyaaaah!^What is this?! A Hylian?!", None, 'junk'), # sfx: Ruto screaming
+    '1226':                                                     ("For you, the day you read this hint was the most important day of your life.^But for me, it was Tuesday.", None, 'junk'),
```

Comment:
I actually really appreciate having the refs in the comments here, but it's not on all of the ones that are references. I definitely recognize a few. Should we make more of an effort to annotate them?